<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>300se control</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Perfomance Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_silent">
       <property name="text">
        <string>Silent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_default">
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_power">
       <property name="text">
        <string>hi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_turbo">
       <property name="text">
        <string>turbo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>fan control</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_cpu_fan">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>cpu fan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_mode">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fan_auto">
         <property name="text">
          <string>auto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fan_max">
         <property name="text">
          <string>max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fan_user">
         <property name="text">
          <string>manual</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="3" alignment="Qt::AlignHCenter">
      <widget class="QScrollBar" name="gpu_fan">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" alignment="Qt::AlignHCenter">
      <widget class="QScrollBar" name="cpu_fan">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="invertedControls">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_gpu_fan">
       <property name="text">
        <string>gpu fan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Keyboard Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rgb_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 128, 128);</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_red">
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(250, 20, 20);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_green">
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(10,255, 10);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_blue">
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(28, 113, 216);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_brightness">
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(222, 221, 218);</string>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="button_save">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
